main/zoe/api/structs.md

```diff
@@ -0,0 +1,44 @@
+# Structs
+
+## EscrowReceiptAndPayout
+
+This is returned by a call to `escrow` on Zoe. An `escrowReceipt` is an ERTP payment which acts as proof of escrow and is normally passed in to a smart contract. The `payout` is a promise that resolves to an array of payments.
+
+```js
+someEscrowReceiptAndPayout: {
+  escrowReceipt: somePayment,
+  payout: someArrayOfPayments,
+}
+```
+
+## OfferRules
+
+```js
+someOfferRules: {
+  payoutRules: [payoutRule1, payoutRule2, payoutRule3],
+  exitRule: someExitRule,
+}
+```
+
+## PayoutRules
+
+`payoutRules` are an array of `PayoutRule`. The possible kinds are 'offerExactly', 'offerAtMost', 'wantExactly', and  'wantAtLeast'.
+
+```js
+somePayoutRules: {
+  kind: 'offerExactly',
+  units: 8,
```

Comment:
This looks really good. Units here should be a labeled extent, which we don't necessarily have to define as a whole object, but we can use a variable name, like `8bucks`

Reply:
Ok, I added the part about the units to the definition and changed the example to use a variable name.